releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -259,9 +259,9 @@ private static List<ReleaseNotesMessage> createAllNotes() throws Exception {
             createReleaseNotesMessage(123, "Mock issue title 3", "Author 5"),
             createReleaseNotesMessage(123, "Mock issue title 4", "Author 6, Author 7"),
             createReleaseNotesMessage(123, "Mock issue title 5 ==> test", "Author 6, Author 7"),
-            createReleaseNotesMessage("Mock issue title 6 L12345678901234567890123456789012345678"
-                + "90123456789012345678901234567890oooooooooooooooooooooooooooooooooooooooooooooo"
-                + "ooong'\"", "Author 1"));
```

Comment:
Why are you removing the long line? This was specifically added if we had one long word and how it would be handled.

Reply:
Changes Done